framework/integration/testresourceset/testresource/resource.go

```diff
@@ -0,0 +1,46 @@
+// +build k8srequired
+
+package testresource
```

Comment:
I would like to not put this package inside the `testresourceset` package. Let's let them live in the same place and on the same level.

Reply:
Done.